CMakeLists.txt

```diff
@@ -271,7 +271,7 @@ target_link_libraries(infinisim PRIVATE QCBOR)
 # check version number of installed node package for minimum required
 find_program(NODE_EXE "node" NO_CACHE QUIET)
 if(NODE_EXE)
-  exec_program("${NODE_EXE}" ARGS "--version" OUTPUT_VARIABLE NODE_VERSION)
+  execute_process(COMMAND "${NODE_EXE}" "--version" OUTPUT_VARIABLE NODE_VERSION)
```

Comment:
Do we not need the `ARGS`?

Reply:
Not according to [the documentation](https://cmake.org/cmake/help/latest/command/execute_process.html#command:execute_process).
But I thought the same initially 😄

Reply:
Alright!